Octokit/Models/Request/RequestParameters.cs

```diff
@@ -49,6 +49,9 @@ static List<PropertyParameter> GetPropertyParametersForType(Type type)
             Justification = "GitHub API depends on lower case strings")]
         static Func<PropertyInfo, object, string> GetValueFunc(Type propertyType)
         {
+            // get underlying type if nullable
+            propertyType = Nullable.GetUnderlyingType(propertyType) ?? propertyType;
```

Comment:
Is this line necessary?  The parameter array was still created correctly for me, even without this line...